### PR TITLE
feat(release-health): Add get session terms function

### PR DESCRIPTION
**Pull request by priscilawebdev**
closes: https://app.asana.com/0/1182975495223897/1199086736853686

### CODE REVIEW

File: src/sentry/static/sentry/app/views/releases/utils/sessionTerm.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        crashed: t(
+          "During session an error with mechanism.handled===false occured which is 'onerror' on 'unhandledrejection' handler"
+        ),
+        abnormal: t('Non applicable for Javascript'),
+        healthy: t('No errors captured during session life-time'),
+        errored: t(
+          'During the session at least one error occurred that did not bubble up to the global handlers, not resulting in the application loading process crashing.'
+        ),
+        unhandled:
+          'En error bubbled up to the global onerror or onunhandledrejection handler',
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/releases/utils/sessionTerm.tsx:82]:
```suggestion
          "En error bubbled up to the global 'onerror' or 'onunhandledrejection' handler",
```

File: src/sentry/static/sentry/app/views/releases/utils/sessionTerm.tsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        errored: mobileTermsDescription.errored,
+        unhandled: mobileTermsDescription.unhandled,
+      };
+  }
+}
+
+type Term = keyof ReturnType<typeof getTermDescriptions>;
+
+export function getSessionTermDescription(term: Term, platform: PlatformKey) {
+  return getTermDescriptions(platform)[term];
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/releases/utils/sessionTerm.tsx:112]: Is there a chance that a `term` gets requested that doesn't exist?

**Review by billyvg — APPROVED** [review]
Can we add a few simple tests?

File: src/sentry/static/sentry/app/views/releases/utils/sessionTerm.tsx (hunk 2 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/views/releases/utils/sessionTerm.tsx:112]: if the term doesn't exist `undefined` will be returned and that is fine. I will add a couple of tests 😉

File: src/sentry/static/sentry/app/views/releases/utils/sessionTerm.tsx [hunk 3]
```diff
@@ -0,0 +1,113 @@
+import {PlatformKey} from 'app/data/platformCategories';
+import {t} from 'app/locale';
+
+const commonTermsDescription = {
+  crashes: t('Number of sessions with a crashed state'),
+  'crash-free-users': t('Number of unique users with non-crashed sessions'),
+  'crash-free-sessions': t('Number of non-crashed sessions'),
```
Comment (matejminar) [inline src/sentry/static/sentry/app/views/releases/utils/sessionTerm.tsx:7]: Should this be a `number` or rather `percentage`?

File: src/sentry/static/sentry/app/views/releases/utils/sessionTerm.tsx [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+        ),
+      };
+    case 'android':
+    case 'cordova':
+    case 'react-native':
+    case 'flutter':
+      return {
+        ...commonTermsDescription,
+        ...mobileTermsDescription,
+        crashed: t('An unhandled exception that result in the application crashing'),
```
Comment (matejminar) [inline src/sentry/static/sentry/app/views/releases/utils/sessionTerm.tsx:59]:
```suggestion
        crashed: t('An unhandled exception that resulted in the application crashing'),
```

File: src/sentry/static/sentry/app/views/releases/utils/sessionTerm.tsx [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+        crashed: t(
+          "During session an error with mechanism.handled===false occured which is 'onerror' on 'unhandledrejection' handler"
+        ),
+        abnormal: t('Non applicable for Javascript'),
+        healthy: t('No errors captured during session life-time'),
+        errored: t(
+          'During the session at least one error occurred that did not bubble up to the global handlers, not resulting in the application loading process crashing.'
+        ),
+        unhandled:
+          "En error bubbled up to the global 'onerror' or 'onunhandledrejection' handler",
```
Comment (matejminar) [inline src/sentry/static/sentry/app/views/releases/utils/sessionTerm.tsx:82]:
```suggestion
          "An error bubbled up to the global 'onerror' or 'onunhandledrejection' handler",
```

File: src/sentry/static/sentry/app/views/releases/utils/sessionTerm.tsx (hunk 3 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/views/releases/utils/sessionTerm.tsx:7]: I will address this in a follow-up PR ;-)

**Pull request merged**